types/slickgrid/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
this is from `yarn add @types/slickgrid` but everything is outdated and 6pac don't have the type for their new version of 6pac. we have to manually maintain this for now

Reply:
I see. Good catch.